apps-rendering/cdk/lib/mobile-apps-rendering.ts

```diff
@@ -87,21 +60,14 @@ export class MobileAppsRendering extends GuStack {
 				InstanceSize.MICRO,
```

Comment:
I'm puzzled why this says `MICRO` when on `main` it's now `SMALL`? Is it possible we'll want to do a rebase/merge `main` at some point?

Reply:
I think because the changes on this branch were made before the instance size change on `main`. It's possibly worth merging `main`, or using `SMALL` in here as well